examples/test_compare_new_engine_wrap.py

```diff
@@ -0,0 +1,10 @@
+from objection_engine.renderer import render_comment_list
+from objection_engine.beans.comment import Comment
+from time import time
+
+comments = [
+    Comment(user_name='Phoenix right', text_content='I am going to slam the desk! I just did it! Did you see that? Was i cool?'),
```

Comment:
Does this properly work? How do you control the desk slam?

Reply:
Thanks for catching this! I think this was an old test file - it was meant to generate a video with the old engine, for comparison to a video generated with the new engine. The equivalent script for the new engine appears to be gone now, so it'd probably make sense to delete this script.